include/igl/opengl/MeshGL.h

```diff
@@ -40,7 +40,10 @@ class MeshGL
     DIRTY_MESH           = 0x00FF,
     DIRTY_OVERLAY_LINES  = 0x0100,
     DIRTY_OVERLAY_POINTS = 0x0200,
-    DIRTY_ALL            = 0x03FF
+    DIRTY_VID_LABELS     = 0x0300,
```

Comment:
This should go 1, 2, 4, 8, no 3.

Reply:
Fixed. Out of curiousity is that the numbering scheme?

Reply:
This is a bitset flag, so you can do bitwise operations like `DIRTY_OVERLAY_LINES & DIRTY_OVERLAY_POINTS` for example.

Reply:
Oh I understand, this makes sense thanks :-)